### PR TITLE
Removes outsider GP Field Office spawn

### PR DESCRIPTION
**Pull request by Spoopy11**
## About The Pull Request
Does what it says in the Title, removes the Outsider spawn from the rotation.  The difficulty scale of this particular spawn is 90 percent death, bringing meaningless rounds to anyone who spawns there who wishes to do character/gimmicks and or interaction with the larger server in general.

Its this or trying to balance GP out for an outsider and that's not how Soj works.

## Changelog
:cl:
Removes Outsider GP Field Office Spawn
Adds two Deep Forest underground spawns for Outsiders
:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Why not just put them somewhere less ouchie?

**Comment by Spoopy11** [comment]
> Why not just put them somewhere less ouchie?

There's plenty of other non-ouchy spawns, this just removes the one spawn which is catastrophically bad and viable when GP didn't have mechs in the regular spawn.  

Plus there's hardly stuff to craft to fix yourself, defend yourself, shoot things from a distance.  It's a high-tech office, so it just makes sense to remove it in general.

**Comment by cdb-is-not-good** [comment]
Ehhh, the spawn could be moved to be somewhere in the field offices that is less immediately and invariably deadly without getting rid of it.

**Comment by ThePainkiller** [comment]
Honestly the worst part of it, and I agree with Das, is literally being fucked over by **GREYSON MECH SPAWNS** at certain places, which are tanky as fuck and hard to cheese, otherwise literal skill issue. I've had to do pretty optimal, streamlined runs to avoid combat most people can't come up with and that's why they die. But you could also argue "they're supposed to be hard!1!!" even if the strategy to overcome it must be a very specific tactic, and people bemoan dying too much to it.
If anything, I find the Forgotten bunker an even more unforgiving experience, specially if you get unlucky with a seeker spawn right outside the door you spawn in, the abundance of spiders, and the deadlyness of the heavy lasers the bots have, only to come out not on the surface but on the CAVES, yet another gauntlet to overcome.

**Comment by cdb-is-not-good** [comment]
Sure, fine- if you're going to remove an outsider spawn I still want a new one put in its place. Just pick some spot that'll be a challenge but not impossible, rather than reducing total spawns by 1

**Comment by Spoopy11** [comment]
I have added two spawns to satisfy that:

https://i.imgur.com/OGSJBlN.png
https://i.imgur.com/yVkfzHQ.png

**Comment by ThePainkiller** [comment]
> I have added two spawns to satisfy that:
> 
> https://i.imgur.com/OGSJBlN.png
> 
> https://i.imgur.com/yVkfzHQ.png

The GP spawn is pretty good, still within "greysons" offices, just not as hard, while also being near the forgotten bunker. A different start.

The Void wolves base, one I'll vote against, for the following reasons:

1. There is already one spawn on the caves, this would make three spawns total in JUST the caves (unless you're moving the current spawn to one of these). The more diverse scenarios, the better.
2. **Too easy**. Hear me out, it's basically the already pre-existing caves spawn's goal: To reach the void wolf base and gain the pre-bottled chems for healing + high end weapons and armor, just this time you don't have to fight through wurms, possible nightmare spawns, and drones, excavating through the caves' gauntlet to reach there fighting tooth and nail. This eliminates the entire struggle from the usual cave spawn by virtue of spawning right next to your best goal. It's very convenient, yes, but feels like cheating.
3. Roundstart outsider managing to succesfully raid Void Wolf base would cuck the shit out of prospies on one of their favorite spots to raid, where it would take them quite the time for prep and travel to get there and achieve the same with almost the same equipment. I'm pretty sure prospectors will complain a lot about this one for this reason alone.

**Comment by cdb-is-not-good** [comment]
I kinda agree. But I did want to buff outsiders so maybe we should give them an easy mode spawn

File: maps/_DeepForest/map/_Deep_Forest.dmm [hunk 1]
```diff
@@ -12542,6 +12542,9 @@
 /obj/structure/closet/crate,
 /turf/simulated/floor/tiled/derelict/white_small_edges,
 /area/nadezhda/outside/one_star)
+"Vx" = (
+/turf/simulated/wall/untinted/onestar_reinforced,
+/area/asteroid/rogue)
```
Comment (Trilbyspaceclone) [inline maps/_DeepForest/map/_Deep_Forest.dmm:12547]: why is the area this?

Comment (Spoopy11) [reply maps/_DeepForest/map/_Deep_Forest.dmm:12547]: Greyson
image

Comment (Trilbyspaceclone) [reply maps/_DeepForest/map/_Deep_Forest.dmm:12547]: Ah, fair enuff

**Comment by Trilbyspaceclone** [comment]
Kicked bots to see if they worked (they do!)

**Comment by ThePainkiller** [comment]
> I kinda agree. But I did want to buff outsiders so maybe we should give them an easy mode spawn

~~They already have the swamp~~

A good thing that'd be a "buff" to outsiders is, like I asked before on development, for the scavenged armor to not cost platinum. Craftable stuff is always a decent "buff" to outsiders without making them have an actual advantage.

**Comment by cdb-is-not-good** [comment]
> > I kinda agree. But I did want to buff outsiders so maybe we should give them an easy mode spawn
> 
> ~They already have the swamp~
> 
> A good thing that'd be a "buff" to outsiders is, like I asked before on development, for the scavenged armor to not cost platinum. Craftable stuff is always a decent "buff" to outsiders without making them have an actual advantage.

Just make the salvaged void lol

**Pull request merged**